Python/Product/Analyzer/Intellisense/AnalysisProtocol.cs

```diff
@@ -851,5 +851,50 @@ internal class UnhandledExceptionEvent : Event {
             }
             public override string name => Name;
         }
+
+        public sealed class GetTestCasesRequest : Request<GetTestCasesResponse> {
+            public const string Command = "getTestCases";
+
+            public override string command => Command;
+
+            public string filename;
```

Comment:
Yep, this is all an internal implementation detail


---
In reply to: [67382713](https://github.com/Microsoft/PTVS/pull/1337#discussion_r67382713) [](ancestors = 67382713)